src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -243,6 +243,7 @@ public override async ValueTask<IReadOnlyList<BulkStateItem>> GetBulkStateAsync(
         {
             var protoHTTPExtension = new Autogenerated.HTTPExtension();
             var contentType = "";
+            Metadata headers = null;
```

Comment:
Can we just move this initialization to line 262 below.

Reply:
The declaration of `headers` cannot be moved into the if-statement, because it's also used in the call to `MakeGrpcCallHandleError` on line 292